whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapResponseJsonTest.java

```diff
@@ -494,6 +495,7 @@ private JsonFactory createJsonFactory() {
 
         objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
         objectMapper.configure(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS, true);
+        objectMapper.configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);
```

Comment:
This call is deprecated -- replace with...

JsonMapper.builder().configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);